ext/tags.go

```diff
@@ -83,18 +96,67 @@ var (
 	HTTPStatusCode = uint16TagName("http.status_code")
 
 	//////////////////////////////////////////////////////////////////////
+	// DB Tags
+	//////////////////////////////////////////////////////////////////////
+
+	// DBInstance is database instance name.
+	DBInstance = stringTagName("db.instance")
+
+	// DBStatement is a database statement for the given database type.
```

Comment:
let's clarify that it can be a query or a prepared statement (i.e., before substitution)